varcode/util.py

```diff
@@ -48,45 +48,52 @@ def random_variants(
 
     variants = []
 
-    while len(variants) < count:
-        transcript_id = rng.choice(transcript_ids)
-        transcript = ensembl.transcript_by_id(transcript_id)
+    # we should finish way before this loop is over but just in case
+    # something is wrong with PyEnsembl we want to avoid an infinite loop
```

Comment:
The one that occurred to me when Ensembl 83 had bad identifier names for transcripts so none of Transcript objects had valid `sequence` fields.